Missing (peer) dependency "shelljs"

got this error when just installing and using download:

```SH
Error: Cannot find module 'shelljs'
Require stack:
- /home/user/project/node_modules/gitly/lib/utils/download.js
- /home/user/project//node_modules/gitly/lib/utils/gitly.js
- /home/user/project/node_modules/gitly/lib/main.js
```

in your package.json shelljs is only dev dep, so i guess it should be prod dep

--- a/package.json
+++ b/package.json
@@ -12,6 +12,7 @@
   "dependencies": {
     "axios": "^1.6.7",
     "cross-spawn": "^7.0.3",
+    "shelljs": "^0.8.5",
     "tar": "^6.2.0"
   },
   "devDependencies": {
@@ -28,7 +29,6 @@
     "eslint-plugin-prefer-arrow": "^1.2.3",
     "jest": "^29.7.0",
     "prettier": "^3.2.5",
-    "shelljs": "^0.8.5",
     "ts-jest": "^29.1.2",
     "tslib": "^2.6.2",
     "typescript": "^5.3.3"

